main/lp/learnpath.class.php

```diff
@@ -11157,9 +11157,9 @@ public function isFirstOrLastItem($currentItemId)
         $lpItemId = [];
 
         $typeListNotToVerify = self::getChapterTypes();
-        foreach ($this->items as $item) {
-            if (!in_array($item->get_type(), $typeListNotToVerify)) {
-                $lpItemId[] = $item->get_id();
+        foreach ($this->get_toc() as $item) {
```

Comment:
A small comment would be nice (like "//Using get_toc() is equivalent to using $this->items because X and Y")